pkg/services/sqlstore/migrations/migrations.go

```diff
@@ -87,6 +87,12 @@ func (*OSSMigrations) AddMigration(mg *Migrator) {
 			addCommentMigrations(mg)
 		}
 	}
+
+	if mg.Cfg != nil && mg.Cfg.IsFeatureToggleEnabled != nil {
+		if mg.Cfg.IsFeatureToggleEnabled(featuremgmt.FlagPanelTitleSearch) {
+			addEntityEventsTableMigration(mg)
+		}
+	}
```

Comment:
Similar to https://github.com/grafana/grafana/pull/48095, I'd recommend not putting migrations behind feature toggles

Reply:
this flag is dev-mode only so I added the condition: https://github.com/grafana/grafana/blob/3e7db088acd1da32ea489b0756cdf4e15ec2da69/pkg/services/featuremgmt/registry.go#L74-L77